cmd/kured/main.go

```diff
@@ -265,9 +277,9 @@ func commandReboot(nodeID string) {
 		}
 	}
 
-	// Relies on hostPID:true and privileged:true to enter host mount space
-	rebootCmd := newCommand("/usr/bin/nsenter", "-m/proc/1/ns/mnt", "/bin/systemctl", "reboot")
-	if err := rebootCmd.Run(); err != nil {
+	// Relies on hostPID:true and SYS_BOOT to restart machine
+	err := syscall.Reboot(syscall.LINUX_REBOOT_CMD_RESTART)
```

Comment:
I would also wrap with the unprivileged condition, and use the systemctl reboot for privileged ones.

AFAIK, the syscall will only do the reboot, and therefore we might be at least missing a _sync_  in the reboot process, to not corrupt data... WDYT?

Reply:
Good spot on the `sync`. We actually we need it in both cases, so I have amended my PR to include that instead.